ansible/files/bootstrap/consul/post/test/vault-integration.yml

```diff
@@ -1,78 +1,145 @@
-# test admin.team.policy
-#TODO: Use consul cli for this??
-- name: "{{ service }} - post/test/vault-integration - admin token - write key-value to root '/' is forbidden"
+# NB: playbook requires facts
+#  - consul.vault.acl.admin_token
+#  - consul.vault.acl.producer_token
+#  - consul.vault.acl.consumer_token
+
+#######################
+### TESTS policies
+#######################
+# ACL
+- name: "{{ service }} - Debug tokens"
+  debug:
+    msg: "admin token {{ consul.vault.acl.admin_token }}\nproducer token {{consul.vault.acl.producer_token}}\nconsumer token {{consul.vault.acl.consumer_token}}"
+
+- name: "{{ service }} - list acl - admin token - allow"
+  shell: consul acl token list
+  environment:
+    CONSUL_HTTP_TOKEN: "{{ consul.vault.acl.admin_token }}"
```

Comment:
Shoudn't this have a `failed_when` instance?

Reply:
In case of success console return status code 0, which is the default behavior of Ansible shell